bing_image_downloader/bing.py

```
@@ -31,73 +31,82 @@ def resize(url,size: tuple):
         self.download_count = 0
         self.query = query
         self.output_dir = output_dir
         self.adult = adult
         self.filter = filter
         self.verbose = verbose
         self.seen = set()
-        
+        self.progress_callback = progress_callback
+        self.log_callback = log_callback
 
         assert type(limit) == int, "limit must be integer"
```

Ruff (0.14.10)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)